tests/test_parallelwellinfo.cpp

```diff
@@ -118,7 +118,7 @@ BOOST_AUTO_TEST_CASE(ParallelWellComparison)
     std::vector<Opm::ParallelWellInfo> well_info;
     
     for (const auto& wellinfo : pairs) {                   
-        well_info.emplace_back(wellinfo, Opm::ParallelWellInfo::Communication());         
+        well_info.emplace_back(wellinfo, Opm::Parallel::Communication());
```

Comment:
Should probably include `ParallelCommunication.hpp` directly then.

Reply:
done